website_docs/exporting_data.md

```diff
@@ -55,9 +55,6 @@ system it is running on, the cloud provider hosting that operating system instan
 
 ```go
 resources := resource.New(context.Background(),
-	// Builtin detectors provide default values and support
-	// OTEL_RESOURCE_ATTRIBUTES and OTEL_SERVICE_NAME environment variables
-	resource.WithBuiltinDetectors(),
```

Comment:
Perhaps change this to `resource.WithFromEnv(),` to maintain the detection of resource attributes from the environment?

Reply:
Oh, sure! That makes good sense to me.